core/Int.carp

```diff
@@ -62,11 +62,11 @@
 
   (doc to-int "acts as the identity function to implement the interface.")
   (sig to-int (Fn [Int] Int))
-  (defn to-int [a] a)
+  (defn to-int [a] (the Int a))
```

Comment:
Shouldn't be necessary anymore--I snuck this in when I initially was trying out the rule that type variables can't replace variables (this rule is too strict).

Reply:
Done 
56d3b02